cmd/kubeops/internal/handler/dashboardcompat_test.go

```diff
@@ -0,0 +1,200 @@
+package handler
+
+import (
+	"net/http/httptest"
+	"runtime/debug"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/kubeapps/common/response"
+	h3chart "helm.sh/helm/v3/pkg/chart"
+	h3 "helm.sh/helm/v3/pkg/release"
+	h2chart "k8s.io/helm/pkg/proto/hapi/chart"
+	h2 "k8s.io/helm/pkg/proto/hapi/release"
+
+	"testing"
+)
+
+func TestNewDashboardCompatibleRelease(t *testing.T) {
+	type testScenario struct {
+		// Scenario params
+		Description         string
+		Helm3Release        h3.Release
+		Helm2Release        h2.Release
+		MarshallingFunction func(h2.Release) string
```

Comment:
this is always `asResponse`, in which case you plan to change this function?

Reply:
If we add a test scenario where we test `newDashboardCompatibleRelease` against for example `spew.Sdump`, or a hand-written one where for example not all fields are relevant/populated etc.